Polyfills/Canvas/Source/Image.cpp

```diff
@@ -39,6 +41,7 @@ namespace Babylon::Polyfills::Internal
         : Napi::ObjectWrap<NativeCanvasImage>{info}
         , m_runtimeScheduler{JsRuntime::GetFromJavaScript(info.Env())}
         , m_cancellationSource{std::make_shared<arcana::cancellation_source>()}
+        , m_jsThis{Napi::Persistent(info.This().As<Napi::Object>())}
```

Comment:
I'm not sure this is okay to do. Won't this cause the garbage collector to never collect this object?

Reply:
Fix this by using this->Value() instead.  I noticed at least one place where we use a similar pattern in NativeXR, will update that when I make the WebXRImageTracking PR.